src/OSPSuite.Core/Domain/Services/FormulaTask.cs

```diff
@@ -243,7 +243,7 @@ private void updateLumenSegmentReferencingPath(FormulaUsablePath formulaUsablePa
          var pathToContainer = pathAsList.Take(firstIndex).ToList();
          var container = getContainerOrThrow(pathToContainer, usingFormula);
          //Point to our absolute ORGANISM|LUMEN|container path
-         var lumenSegmentPath = new List<string> {ORGANISM, Organ.LUMEN, container.Name};
+         var lumenSegmentPath = new List<string> {model.Root.Name, ORGANISM, Organ.LUMEN, container.Name};
```

Comment:
@Yuri05 I am adding it by hand here because I am constructing a path manually
I am going to have to reorganize this a bit to fix the PKSim issue anyways but once is good